person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
**OPTIONAL** - I suggest that you make your `id` a random number from `1...1000`. This way an id is automatically assigned instead of inputting it manually. Please fix this 🙏 
```ruby
id = Random.rand(1...1000)
```